app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -1561,7 +1561,13 @@ protected function _saveProducts()
                 }
                 $rowScope = $this->getRowScope($rowData);
 
-                $rowData[self::URL_KEY] = $this->getUrlKey($rowData);
+                /**
+                 * Only populate URL KEY if a value has been provided.
+                 * @see https://github.com/magento/magento2/issues/17023
```

Comment:
Comments are good, can you please remove Magento issues links from the code. Let keep it clean.